nosql/1.0/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Jakarta NoSQL 1.0 (under development)"
+date: 2020-07-23
+summary: "Under Development"
```

Comment:
This should be a very short summary of your project.  Just one or two sentences describing what is being delivered.  Most of the Jakarta Specs indicate that they are "part of the Jakarta EE 9 release".  NoSQL can't say that yet.  But, it might be good to highlight that fact with this Summary.  Something like... "Jakarta NoSQL is a standalone Specification with a future goal of becoming part of the Jakarta EE 9 platform."